app/src/modules/tageuro/tageuro.js

```diff
@@ -0,0 +1,191 @@
+                        } else {
+                            reject("Error: something went wrong while fetching cached response");
+                        }
+                    } else { // no cache
+                    
+                        // Declares the API Caller
+                        var apiCaller = require('../../api_caller');
+
+                        var result = encodeURI(values[0]);
+                        var api_request = 'https://www.sentimantic.net/qwant/apiReqIA/image?fid='+result;
+
+                        // Defines the structure of the answer
+                        var structure = {
+                            "image":  "String"
+                        };
+                            
+
+                        // Call the API and get back data
+                        apiCaller.call(api_request, structure, proxyURL, this.timeout, redisTools).then((apiRes) => {
+                             redisTools.saveToCache(CACHE_KEY, apiRes, CACHE_EXPIRE);
+                            apiRes.fromCache = false;
+                            resolve(apiRes);
+                        }).catch((error) => {
+                            reject(error);
+                        });
+                            
+                    }
+                });
+            } else {
+                reject("Couldn't process query.")
+            }
+        });
+    },
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * If your name needs to be translated, use this function getName().
+     * @returns the tab name translated
+     */
+
+    getName: function (i18n) {
+        const _ = i18n._;
+        return _("TagEuro", "tageuro");
+    },
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * Otherwise, if your name doesn't need to be translated, use this attribute.
+     */
+
+    name: "TagEuro",
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * If your keyword needs to be translated, use this function getKeyword().
+     * @returns keyword translated
+     */
+    getKeyword: function (i18n) {
+        const _ = i18n._;
+        var finalKeywords = ""
+        keywords.forEach(function(element) {
+            finalKeywords += element + "|"
+        });
+
+        finalKeywords = finalKeywords.substr(0, oldStr.length-1);
+        return _("Nicolas Dupont(\-| )Aignan", "tageuro");//finalKeywords;
+    },
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * Otherwise, if your keyword doesn't need to be translated, use this attribute.
+     * The keyword can be a regex. If you need help for your regex, use this https://regex101.com/#javascript
+     */
+
+    keyword: "(N|n)icolas (D|d)upont(\-| )(A|a)ignan|(F|f)lorian (P|p)hilippot|(R|r)apha(\ë|e)l (G|g)lucksmann|(J|j)ean (C|c)hristophe (L|l)agarde|(F|f)ran(\ç|c)ois (A|a)sselineau|(I|i)an (B|b)rossat|(J|j)ean (L|l)assalle|(Y|y)annick (J|j)adot|(F|f)ran(\ç|c)ois (X|x)avier (B|b)ellamy|(N|n)athalie (L|l)oiseau|(J|j)ordan (B|b)ardella|(M|m)anon (A|a)ubry|(N|n)athalie (A|a)rthaud",
```

Comment:
You have set the "flag" field with i for insensitive, so you don't need to add upper-case and lower-case letters.
```suggestion
    keyword: "nicolas\s+dupont(\-|\s+)aignan ....",
```